modules/installation-creating-aws-vpc-subnets-edge.adoc

```diff
@@ -29,32 +35,32 @@ If you do not use the provided CloudFormation template to create your AWS infras
 +
 [source,terminal]
 ----
-$ aws cloudformation create-stack --stack-name <stack_name> \ <1>
+$ aws cloudformation create-stack --stack-name <stack_name> \// <1>
   --region ${CLUSTER_REGION} \
-  --template-body file://<template>.yaml \ <2>
+  --template-body file://<template>.yaml \// <2>
   --parameters \
-    ParameterKey=VpcId,ParameterValue="${VPC_ID}" \ <3>
-    ParameterKey=ClusterName,ParameterValue="${CLUSTER_NAME}" \ <4>
-    ParameterKey=ZoneName,ParameterValue="${ZONE_NAME}" \ <5>
-    ParameterKey=PublicRouteTableId,ParameterValue="${ROUTE_TABLE_PUB}" \ <6>
-    ParameterKey=PublicSubnetCidr,ParameterValue="${SUBNET_CIDR_PUB}" \ <7>
-    ParameterKey=PrivateRouteTableId,ParameterValue="${ROUTE_TABLE_PVT}" \ <8>
+    ParameterKey=VpcId,ParameterValue="${VPC_ID}" \// <3>
+    ParameterKey=ClusterName,ParameterValue="${CLUSTER_NAME}" \// <4>
+    ParameterKey=ZoneName,ParameterValue="${ZONE_NAME}" \// <5>
+    ParameterKey=PublicRouteTableId,ParameterValue="${ROUTE_TABLE_PUB}" \// <6>
+    ParameterKey=PublicSubnetCidr,ParameterValue="${SUBNET_CIDR_PUB}" \// <7>
+    ParameterKey=PrivateRouteTableId,ParameterValue="${ROUTE_TABLE_PVT}" \// <8>
     ParameterKey=PrivateSubnetCidr,ParameterValue="${SUBNET_CIDR_PVT}" <9>
```

Comment:
The subnet labels is also missing. It will help to identify the outpost subnets in the tag:Name
```suggestion
    ParameterKey=PrivateSubnetCidr,ParameterValue="${SUBNET_CIDR_PVT}" \// <9>
    ParameterKey=PrivateSubnetLabel,ParameterValue="private-outpost" \//
    ParameterKey=PublicSubnetLabel,ParameterValue="public-outpost" \//
    ParameterKey=OutpostArn,ParameterValue="${OUTPOST_ARN}" <X>
```